chore: actl migration

gochicus opened the pull request:
affects: @lessons/how-to-js, @lessons/how-to-use-functions, @lessons/how-to-use-objects,
@lessons/loops-and-conditions, @practice/actions-with-page, @practice/events-in-page,
@practice/food-delivery, @practice/refactor, @practice/web-diary

ISSUES CLOSED: #61

gochicus committed:
chore: actl migration

affects: @lessons/how-to-js, @lessons/how-to-use-functions, @lessons/how-to-use-objects,
@lessons/loops-and-conditions, @practice/actions-with-page, @practice/events-in-page,
@practice/food-delivery, @practice/refactor, @practice/web-diary

ISSUES CLOSED: #61

TorinAsakura commented on package.json:
```diff
@@ -2,32 +2,23 @@
   "private": true,
   "scripts": {
-    "lint": "./node_modules/.bin/eslint --fix './**/*.{ts,js,tsx,jsx}'"
+    "lint": "actl lint --fix",
```
изучи исходники внимательно и не делай больше ненужных телодвижений

TorinAsakura commented on package.json:
```diff
@@ -2,32 +2,23 @@
   },
   "devDependencies": {
+    "@atlantis-lab/actl": "^0.4.32",
     "@atlantis-lab/tsconfig": "^0.1.2",
```
тсконфиг тут зачем?

TorinAsakura commented on .github/workflows/actlcheck.yml:
```diff
@@ -1,9 +1,9 @@
-name: lintcheck
+name: actlcheck
 
 on: pull_request
 
 jobs:
-  lintcheck:
+  actlcheck:
```
зачем? если это тот же линтер?

gochicus committed:
refactor: refactor according to lint rules

affects: @lessons/how-to-js, @lessons/how-to-use-functions, @lessons/how-to-use-objects,
@lessons/loops-and-conditions, @practice/events-in-page, @practice/food-delivery, @practice/refactor

ISSUES CLOSED: #61

TorinAsakura commented on package.json:
```diff
@@ -2,32 +2,22 @@
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "actl precommit"
     }
```
где commit-msg?
